pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -546,6 +546,18 @@ form {
       width: 100%;
     }
   }
+
+  .form-group {
```

Comment:
Can you add comments that gives more context for where these classes are used since they aren't in the Pegasus repo?